_includes/navbar.html

```diff
@@ -1,10 +1,12 @@
 <header class="nav-bar">
     <div class="nav-content">
-        <a href="{{ '/' | relative_url }}">
-            <div class="nav-logo">
-                <img src="./assets/img/logo.svg" />
-                <a class="nav-link" href="{{ site.url }}/cv/resume.pdf">Resume</a>                
-            </div>
-        </a>        
+        <div class="nav-logo">
+            <a href="{{ '/' | relative_url }}"><img src="./assets/img/logo.svg" /></a>    
+        </div>
+        <a class="nav-link" href="{{ site.url }}/projects">Projects</a>
+        <a class="nav-link" href="{{ site.url }}/experience">Work Experience</a>
+        <a class="nav-link" href="{{ site.url }}/education">Education</a>
+        <a class="nav-link" href="{{ site.url }}/achievements">Achivements</a>
```

Comment:
Also a typo to fix here

```suggestion
        <a class="nav-link" href="{{ '/achievements' | absolute_url }}">Achievements</a>
```